packages/bot-web-ui/src/components/dashboard/quick-strategy/quick-strategy-components/form.tsx

```diff
@@ -86,9 +86,15 @@ const QuickStrategyForm = ({
                                     'quick-strategy__form-content--safari-fix': isSafari(),
                                 })}
                             >
-                                <div className='quick-strategy__title'>{localize('Quick strategy')}</div>
-                                <div className='quick-strategy__description'>
-                                    {localize('Choose a template and set your trade parameters.')}
+                                <div className='quick-strategy__text'>
+                                    <Text size='sm' weight='bold' line_height='s'>
+                                        {localize('Quick strategy')}
+                                    </Text>
+                                </div>
+                                <div className='quick-strategy__text'>
+                                    <Text size='sm' weight='normal' line_height='m'>
```

Comment:
default line height is m no need to pass for text component.

Reply:
@rupato-deriv ok, thanks 👌